modules/page_base.py

```diff
@@ -131,17 +137,22 @@ def get_element(self, name: str, *label) -> WebElement:
         selenium.webdriver.remote.webelement.WebElement
             The WebElement object referred to by the element dict.
         """
+        logging.info("====")
+        logging.info(f"Getting element {name}")
         if "seleniumObject" in self.elements[name]:
+            logging.info("Returned from object cache!")
             return self.elements[name]["seleniumObject"]
         element_data = self.elements[name]
         selector = self.get_selector(name, *label)
         if "shadowParent" in element_data:
+            logging.info(f"Found shadow parent {element_data['shadowParent']}...")
             shadow_parent = self.get_element(element_data["shadowParent"])
             shadow_element = self.utils.find_shadow_element(shadow_parent, selector)
             self.elements[name]["seleniumObject"] = shadow_element
             return shadow_element
         found_element = self.driver.find_element(*selector)
         self.elements[name]["seleniumObject"] = found_element
+        logging.info(f"Returning element {name}.\n")
```

Comment:
This was the only one with a trailing newline (not that there is anything wrong/bad with that, just calling it out).

Reply:
Since this is the end of the selector-grabbing phase, the newline is there to separate each phase.